markdown/org/docs/designs/shelly/instructions/en.md

```diff
@@ -0,0 +1,92 @@
+
+   - T-shirt
+     - The author hasn't made one yet. If someone knows what eases to use, please let me know.
+	 - Eases will be quite a bit more larger (more positive) than for other shirt types.
+
+   - Sweatshirt
+     - The author hasn't made one yet. If someone knows what eases to use, please let me know.
+	 - Eases will be similar to that for a t-shirt, or a little larger.
+
+ - The neckband length will be mostly determined by the material of the neckband. Stretch fabric will generally use a length of around 80%. Less stretchy fabric will be longer, but always shorter than 100%, and ribbed knit (traditional t-shirt neckbands) are around 60-70%.
+
+### Step 1: Forming the body and attaching the sleeves
+
+- With _good sides together_, sew the front piece to each sleeve along the raglan seam (the diagonal seam running from the neck to the armpit).
+- With _good sides together_, sew the back piece to each sleeve along the raglan seam. You should now have a single piece with a circular neck hole in the middle.
+
+### Step 2: Attaching the neckband
+
+- With _good sides together_, fold the neckband in half long ways (so it's half as long) and sew it into a loop.
+- With _wrong sides together_, fold the neckband in half short ways (so it's half as wide).
+- Turn the shirt right-side-out
+- Mark the quarter points on your neckband with ball point pins, clips, or chalk.
+- Do the same around the neck hole, marking the center of the front, the center of the back, and the midpoints on either sleeve.
+- With _good sides together_, line up the unfinished edges of the neckband with the edge of the neck hole, and line up the quarter points of the neckband with the quarter points of the neck hole. Pin or clip the neckband around the neck hole.
+ - Sew carefully around the neck hole, making sure to sew through all 3 layers.
+ - Turn the neckband over. It should lay flat.
+
+### Step 3: Closing the sleeves and sides
+
+- Turn the shirt inside-out.
+- With _good sides together_, place the front and back pieces together and pin/clip their sides together.
+- With _good sides together_, close the sleeve and pin/clip it along the sleeve seam.
+- You should have a single seam prepared, going from the end of the sleeve to the bottom hem of the body.
+- Sew it shut.
+- Repeat for the other side.
+
+### Step 4: (Optional) Hemming
+
+- Hem the sleeves and/or the bottom of the shirt using a single-fold hem. Using a twin needle will add some stretch to the stitching, but in either case the use of a stretch stitch is still recommended for high-stretch fabrics.
+- Alternatively, you can overlock or serge the raw edge _without_ folding it over to give it a little bit of substance without adding as much bulk as a folded hem, or you can make a rolled hem.
+- Or you can leave the edges unfinished, particularly if it's swim fabric or some other fabric that does not fray at all.
```

Comment:
Please add a Step 5: Enjoy!
This is a fun little thing we like to add at the end of our instructions. It can be fun and cheeky or it can simply be "You're all done now enjoy your new shirt!", See, Hi, Albert, Holmes, Octoplushy for more playful examples.

Reply:
Needs more info.

Reply:
Done

Reply:
Oh I like that signoff!